### PR TITLE
Renaming work

### PR DESCRIPTION
**Pull request by whitej6**

### CODE REVIEW

File: docs/admin/uninstall.md [hunk 1]
```diff
@@ -8,4 +8,4 @@ Remove the configuration you added in `nautobot_config.py` from `PLUGINS` & `PLU
 
 ## Database Cleanup
 
-Drop all tables from the plugin: `nautobot_plugin_bgp_models*`.
+Drop all tables from the app: `nautobot_bgp_models*`.
```
Comment (glennmatthews) [inline docs/admin/uninstall.md:11]: Can this be changed to a `nautobot-server migrate nautobot_bgp_models zero` or does that not work for this app at present?

File: docs/dev/models.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ The data models introduced by the BGP plugin support the following Nautobot feat
 - Webhooks
 
 !!! note
-    The data models defined in this plugin were inspired by the [Openconfig BGP data model](https://yangcatalog.org/api/services/tree/openconfig-bgp@2021-06-16.yang) and the [RFC 9234](https://datatracker.ietf.org/doc/rfc9234/).
+    The data models defined in this app were inspired by the [Openconfig BGP data model](https://yangcatalog.org/api/services/tree/openconfig-bgp@2021-06-16.yang) and the [RFC 9234](https://datatracker.ietf.org/doc/rfc9234/).
```
Comment (glennmatthews) [inline docs/dev/models.md:30]:
```suggestion
    The data models defined in this app were inspired by the [Openconfig BGP data model](https://yangcatalog.org/api/services/tree/openconfig-bgp@2021-06-16.yang) and [RFC 9234](https://datatracker.ietf.org/doc/rfc9234/).
```

File: docs/user/app_getting_started.md [hunk 3]
```diff
@@ -8,11 +8,11 @@ To install the App, please follow the instructions detailed in the [Installation
 
 ## First steps with the App
 
-We will now explore the workflows for the two most common BGP Plugin use cases: modeling internal and external peerings. We will showcase how to create each type of peering with the minimum number of steps and inputs required from users.
+We will now explore the workflows for the two most common BGP App use cases: modeling internal and external peerings. We will showcase how to create each type of peering with the minimum number of steps and inputs required from users.
 
 ### Menu Item
 
-All of the "Add object" actions are available under BGP Models Plugin Menu in the top navigation bar:
+All of the "Add object" actions are available under BGP Models App Menu in the top navigation bar:
```
Comment (glennmatthews) [inline docs/user/app_getting_started.md:15]:
```suggestion
All of the "Add object" actions are available under the Routing menu in the top navigation bar:
```

File: nautobot_bgp_models/api/filter_backends.py [hunk 4]
```diff
@@ -1,4 +1,4 @@
-"""Filter Backeds in use by BGP models plugin."""
+"""Filter Backeds in use by BGP models app."""
```
Comment (glennmatthews) [inline nautobot_bgp_models/api/filter_backends.py:1]:
```suggestion
"""Filter Backends in use by BGP models app."""
```

**Review by glennmatthews — APPROVED** [review]

File: docs/admin/uninstall.md (hunk 1 above)
Comment (whitej6) [reply docs/admin/uninstall.md]: Updated full file to current doc pattern

**Pull request merged**